### PR TITLE
Tidy HTML

### PR DESCRIPTION
**Pull request by RyanZim**
As per #6.

I also added content to meta description tag.

@mde Shouldn't something be done about `<meta name="author" content="">`?

### CODE REVIEW

**Comment by RyanZim** [comment]
@mde ping?

**Comment by RyanZim** [comment]
@mde ping?

**Comment by RyanZim** [comment]
@mde ping?

File: index.html [hunk 1]
```diff
@@ -405,12 +367,19 @@ <h2 class="title text-center">License</h2>
...
+        i[r] = i[r] || function() {
+          (i[r].q = i[r].q || []).push(arguments)
+        }, i[r].l = 1 * new Date();
+        a = s.createElement(o),
+          m = s.getElementsByTagName(o)[0];
+        a.async = 1;
+        a.src = g;
+        m.parentNode.insertBefore(a, m)
+      })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
       ga('create', 'UA-59448014-1', 'auto');
```
Comment (TimothyGu) [inline index.html:374]: I'd rather not change this block. The compressed script comes straight from Google, and it's not like we need readability here.

Comment (mde) [inline index.html:374]: Good call. Can we leave this as-is?

**Comment by mde** [comment]
@RyanZim Sorry for the delay on this. I'll merge as soon as the GA code is back in its ugly blob.

File: index.html (hunk 1 above)
Comment (RyanZim) [inline index.html:374]: Sure, will fix.

**Comment by RyanZim** [comment]
@mde Done.

**Comment by mde** [comment]
Nice work. Thank you, sir!

**Pull request merged**